scripts/eval/find_altloc_selections.py

```
@@ -0,0 +1,68 @@
+    This script will likely change until we settle on a final input/output
+    results directory structure.
+    """
+    output_lines = []
+    warnings = []  # output these all at the end so they're easy to see.
+    for line in open(args.input_csv):
+        if not output_lines:
+            keys = line.strip().split(",")
+            if not all(k in keys for k in ("name", "structure", "resolution", "density")):
+                raise ValueError(
+                    "Input CSV file must keys 'name', 'structure', 'resolution', and 'density'"
+                )
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)